admin/build/createpackage

```diff
@@ -55,10 +57,15 @@ chmod -R 755 $PACKAGING_DIR/DEBIAN
 
 INSTALL_DIR=$PACKAGING_DIR/srv/lucid-piezo-admin-service/bin
 mkdir -p $INSTALL_DIR
-
-cp -r $SERVER_HOME/target/staged  $INSTALL_DIR/staged-$VERSION
+INSTALL_JAR_DIR=$INSTALL_DIR/staged-$VERSION
+mkdir -p $INSTALL_JAR_DIR
+
+#copy jars
+for JAR in `find $SERVER_HOME/lib_managed/ -name "*jar"`; do
+    cp $JAR $INSTALL_JAR_DIR/
+done
+cp $SERVER_HOME/target/scala-2.11/admin_*.jar $INSTALL_JAR_DIR/
```

Comment:
We'll have to make sure this starts off with a clean workspace, else any admin jar build with earlier version numbers will be included and conflict.